packages/@aws-cdk/aws-eks/README.md

```diff
@@ -458,9 +458,18 @@ To disable the installation of the termination handler, set the `spotInterruptHa
 #### Bottlerocket
 
 [Bottlerocket](https://aws.amazon.com/bottlerocket/) is a Linux-based open-source operating system that is purpose-built by Amazon Web Services for running containers on virtual machines or bare metal hosts.
-At this moment, `Bottlerocket` is only supported when using self-managed auto-scaling groups.
 
-> **NOTICE**: Bottlerocket is only available in [some supported AWS regions](https://github.com/bottlerocket-os/bottlerocket/blob/develop/QUICKSTART-EKS.md#finding-an-ami).
+`Bottlerocket` is supported when using managed nodegroups or self-managed auto-scaling groups.
+
+> **NOTICE**: Bottlerocket is supported in all AWS commercial regions. For more details, see [Bottlerocket FAQs](https://aws.amazon.com/bottlerocket/faqs/) and [Bottlerocket Open Source Blog](https://aws.amazon.com/blogs/opensource/announcing-the-general-availability-of-bottlerocket-an-open-source-linux-distribution-purpose-built-to-run-containers/).
```

Comment:
Does this need to be called out as a NOTICE, rather than a normal paragraph? Being supported in all commercial regions is kind of what users expect by default. I would just merge these two paragraphs.

Other than this, the PR looks good to be shipped!

Reply:
> Does this need to be called out as a NOTICE, rather than a normal paragraph? Being supported in all commercial regions is kind of what users expect by default. I would just merge these two paragraphs.
> 
> Other than this, the PR looks good to be shipped!

Agree. Let me adjust this and make it a normal paragraph.

Reply:
DONE